compiler/reader.go

```diff
@@ -121,7 +158,7 @@ func FetchFile(fileurl string) ([]byte, error) {
 	}
 	defer response.Body.Close()
 	if response.StatusCode != 200 {
-		return nil, errors.New(fmt.Sprintf("Error downloading %s: %s", fileurl, response.Status))
+		return nil, fmt.Errorf("Error downloading %s: %s", fileurl, response.Status)
 	}
 	bytes, err = ioutil.ReadAll(response.Body)
 	if fileCacheEnable && err == nil {
```

Comment:
Should the following assignment of `fileCache[fileurl] = bytes` be synchronized?

Reply:
This is unsychronized because it is in a private function.